Documentation/mkfs.btrfs.rst

```diff
@@ -188,6 +188,12 @@ OPTIONS
 
                 $ mkfs.btrfs -O list-all
 
+--compression <algo>
```

Comment:
This should support all algorithms, lzo and zlib can be added later. Also the compression level should be supported, for zstd it may make sense to use the highest level (matching kernel levels) when creating the fs.